ENH: Draft JOSS paper

jvkersch opened the pull request:
Happy new year!

This is a draft for a [JOSS](https://joss.theoj.org/) submission ﻿about vpsearch. I think the paper is submission ready, modulo perhaps some tweaks.

You can get a preview of what the paper will look like at https://whedon.theoj.org/.

stevenjkern reviewed (changes requested):
Looks lovely, @jvkersch! I have just a few very small comments. 🙏🏻 ❤️

jvkersch commented on paper/paper.md:
```diff
@@ -0,0 +1,130 @@
+bacterial classification) VPsearch outperforms both BLAST (7x speedup) and
+ggsearch36 from the FASTA suite (27x speedup) without any loss in accuracy.
+
+The VPsearch tool is implemented in Python, using Cython for
```
Good idea. There's an [IEEE paper ](https://ieeexplore.ieee.org/document/5582062)(from 2011) that is mentioned on the Cython homepage, I'll add that.

jvkersch commented:
@stevenjkern Thanks for picking up on this and for the quick turnaround in reviewing. I was going to send you an email about this, but you saw the PR before I had done so.

I think I've addressed your comments, let me know if this looks good. If you approve of the PR, I'll take that as a sign that the paper is ready for submission to JOSS.

stevenjkern commented on paper/paper.bib:
```diff
@@ -0,0 +1,166 @@
+
+@ARTICLE{2011-behnel-cython,
+  author={Behnel, Stefan and Bradshaw, Robert and Citro, Craig and Dalcin, Lisandro and Seljebotn, Dag Sverre and Smith, Kurt},
+  journal={Computing in Science   Engineering},
```
```suggestion
  journal={Computing in Science \& Engineering},
```

stevenjkern reviewed (approved):
LGTM modulo missing &

:shipit:

jvkersch commented:
Thanks @stevenjkern, good catch! I'll merge the PR now and submit off the draft, and we can open separate PRs for follow-up changes.
